### DISCUSSION TITLE
WorkloadSimulator for V5?

### DISCUSSION THREAD

**Discussion by davidsearle-antithesis** [ravendb/ravendb#15322] [Ideas]
Hi everybody,

I'm looking at using a deterministic simulation testing technology against RavenDB. I see that the workloadsimulator was available until v2.5. Will this still work for V5 and onwards?

Thanks in advance,

Dave

**Comment by garayx** [comment]
Hi,

No, it won't work on V5, 
I guess you can use the benchmarks for this, it coverts most of the RavenDB features, https://github.com/ravendb/ravendb/tree/v5.4/bench

**Comment by davidsearle-antithesis** [comment]
Hi Egor,

Thanks for pointing me in the right direction!



On Thu, 24 Nov 2022 at 08:47, Egor ***@***.***> wrote:

> Hi,
>
> No, it won't work on V5,
> I guess you can use the benchmarks for this, it coverts most of the
> RavenDB features, https://github.com/ravendb/ravendb/tree/v5.4/bench
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ravendb/ravendb/discussions/15322#discussioncomment-4223553>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AT4N6Z57RVUF7S7PUKBXL5DWJ4TTHANCNFSM6AAAAAASI6PVHU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 



David Searle | EMEA Field Operations | Antithesis

+44 (7919) 352561 | ***@***.***

-- 

*-----------------------------*
*This email and any files transmitted with 
it are confidential and intended solely for the use of the individual or 
entity for whom they are addressed. If you received this message in error, 
please notify the sender and remove it from your system.*
